spec/features/categories_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe 'Categories', type: :system, js: true do
+  describe 'index page' do
+    before(:example) do
+      @user = User.create(name: 'Menendez', email: 'menendezon@gmail.com', password: 'anything')
+      sign_in @user
+
+      @category = Category.create(author: @user, name: 'Cheese', icon: 'https://cdn-icons-png.flaticon.com/512/4063/4063291.png')
+      visit categories_path
+    end
+
+    it 'renders category name' do
+      expect(page).to have_content(@category.name)
+    end
+
+    it 'renders category icon' do
+      find("img[src='#{@category.icon}']")
+    end
+
+    it 'renders category\'s total expenses' do
+      expect(page).to have_content(@category.expenses.sum(:amount))
```

Comment:
- [ ] As stated earlier, some of the tests that you've written are failing at the moment. **The only one that passed succesfully were tests written in `/models` directory**, kindly work around this in order to fulfil the project requirement. 🙏